parsing/cparser.py

```diff
@@ -220,12 +219,31 @@ def p_declaration_1(self, t):
 
         print(type(init_declarator_list))
         print(init_declarator_list)
+
         for init_declarator in init_declarator_list:
-            symbol, _ = self.compiler_state.symbol_table.find(init_declarator['declarator'])
-            if not symbol:
-                symbol = Symbol(identifier=init_declarator['declarator'])
+
+            pointers = init_declarator['pointers']
+
+            # Code specific to array declaration
+            if type(init_declarator['declarator']) is dict:
```

Comment:
I think this should be a `dict` anyways. I have the check in there, so the old code is still accessible.